packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue

```diff
@@ -277,6 +276,10 @@ export default mixins().extend({
 		&:focus {
 			border-right: none;
 		}
+
+		&:disabled {
+			cursor: not-allowed !important;
```

Comment:
what if an expression is set? that should be clickable and open expressions modal

Reply:
This is just for the mode selector drop-down. Clicking the input field with an expression will open the modal.

Reply:
oh I see.. thanks for explaining